frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/ConditionalRoutingOptionModal.tsx

```diff
@@ -0,0 +1,259 @@
+import Attachment from '~components/Field/Attachment'
+import FormErrorMessage from '~components/FormControl/FormErrorMessage'
+import { ModalCloseButton } from '~components/Modal'
+import { ProgressIndicator } from '~components/ProgressIndicator/ProgressIndicator'
+
+import { ConditionalRoutingConfig, FieldItem } from './RespondentBlock'
+
+const NUM_STEPS = 2
+
+interface StepOneModalContentProps {
+  stepNumber: number
+  setStepNumber: (step: number) => void
+  isMobile: boolean
+  onDownloadCsvClick: ConditionalRoutingOptionModalProps['onDownloadCsvClick']
+  onClose: ConditionalRoutingOptionModalProps['onClose']
+}
+
+const StepOneModalContent = ({
+  stepNumber,
+  setStepNumber,
+  isMobile,
+  onDownloadCsvClick,
+  onClose,
+}: StepOneModalContentProps) => (
+  <ModalContent minW="fit-content">
+    <ModalCloseButton />
+    <ModalHeader>
+      <Text mb="0.25rem">Add email(s) to options</Text>
+      <ProgressIndicator
+        numIndicators={NUM_STEPS}
+        currActiveIdx={stepNumber}
+        onClick={setStepNumber}
+      />
+    </ModalHeader>
+    <ModalBody>
+      <Stack
+        justifyContent="center"
+        spacing="3rem"
+        direction={isMobile ? 'column' : 'row'}
+      >
+        <Box w={isMobile ? '100%' : '25rem'}>
+          <Stack spacing="0.5rem" mb="2.5rem">
+            <Text textStyle="body-2">
+              We have created a CSV template with the options from the field you
+              selected.{' '}
+              <Text as="span" fontWeight="semibold">
+                Please download the CSV template and add the emails for each
+                option.
+              </Text>
+            </Text>
+            <Button
+              w="100%"
+              leftIcon={<BiDownload fontSize="1.5rem" />}
+              onClick={onDownloadCsvClick}
+            >
+              Download and edit CSV
+            </Button>
+          </Stack>
+          <Stack spacing="1rem">
+            <Text textStyle="subhead-1">How to use the CSV template:</Text>
+            <Box>
+              <Text textStyle="subhead-1">Column A</Text>
+              <Text textStyle="body-2">
+                This contains all the options from your field.{' '}
+                <Text as="span" fontWeight="semibold">
+                  Do not edit, reorder or delete anything in this column.
+                </Text>
+              </Text>
+            </Box>
+            <Box>
+              <Text textStyle="subhead-1">Column B</Text>
+              <Text textStyle="body-2">
+                Add the email(s) to send the form to for each option.{' '}
+                <Text as="span" fontWeight="semibold">
+                  Separate multiple email(s) with a comma.
+                </Text>
+              </Text>
+            </Box>
+          </Stack>
+        </Box>
+        <Stack spacing="1rem" alignItems="center">
+          <Image
+            w="466px"
+            src={'public/static/images/conditional-routing-example.png'}
```

Comment:
We might be able to do something like this instead of inline imports.
```
import routingExample from 'public/static/images/conditional-routing-example.png'
```

Reply:
This indeed makes the import much cleaner, done in [52df545](https://github.com/opengovsg/FormSG/pull/7804/commits/52df54556029ce6b15bac42ba4441d72a997d4ab)